.github/pull_request_template.md

```diff
@@ -5,7 +5,25 @@ Describe in plain language the motivation (bug, feature, etc.) behind the change
 Fixes: #NNN, #NNN, ...
```

Comment:
the `Fixes: #NNN` syntax will not have an issue automatically fixed. maybe we can change this as part of this PR as well? also, for multiple issues, the syntax is `fixes #NNN, fixes #NNN, ...`, in other words each `#NNN` needs to be prefixed with a keyword that triggers closing the issue (`closes`, `fixes`, etc.) more info on [the github docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

Reply:
In practice github does accept the colon (I've always typed it out of habit and never had a problem), I just checked an example issue to make sure I wasn't imagining it :-) (https://github.com/vectorizedio/redpanda/pull/2988)

Reply:
oh, true! my bad, i should have actually tested it

Reply:
i've update to match the format specified in github docs